### PR TITLE
Add Gaussian Blur

### PR DESCRIPTION
**Pull request by LordDarkula**
Add filter UDF abstract classes and Gaussian blur filter.

### CODE REVIEW

File: eva/udfs/pytorch_abstract_udf.py [hunk 1]
```diff
@@ -113,3 +113,92 @@ def __call__(self, *args, **kwargs):
         if isinstance(frames, pd.DataFrame):
             frames = frames.transpose().values.tolist()[0]
         return nn.Module.__call__(self, frames, **kwargs)
+
+
+class PytorchAbstractFilterUDF(AbstractFilterUDF, nn.Module, GPUCompatible, ABC):
```
Comment (gaurav274) [inline eva/udfs/pytorch_abstract_udf.py:118]: Create new abstract files for such filters

File: eva/udfs/gaussian_blur.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        self.padding_mode = padding_mode
+
+    @property
+    def input_format(self) -> FrameInfo:
+        # TODO: figure this out
+        return FrameInfo(-1, -1, 3, ColorSpace.RGC)
+
+    def _get_frames(self, frames: Tensor) -> Tensor:
+
+        model = nn.Conv2d(frames[1], frames[1],
```
Comment (gaurav274) [inline eva/udfs/gaussian_blur.py:51]: Move this to init

File: eva/udfs/abstract_udfs.py [hunk 3]
```diff
@@ -67,3 +67,42 @@ def classify(self, frames: np.ndarray) -> pd.DataFrame:
 
     def __call__(self, *args, **kwargs):
         return self.classify(*args, **kwargs)
+
+
+class AbstractFilterUDF(metaclass=ABCMeta):
```
Comment (gaurav274) [inline eva/udfs/abstract_udfs.py:72]: Move to new file

File: eva/udfs/abstract_udfs.py [hunk 4]
```diff
@@ -67,3 +67,42 @@ def classify(self, frames: np.ndarray) -> pd.DataFrame:
...
+    """
+    Abstract class for UDFs. All the UDFs which apply filters
```
Comment (gaurav274) [inline eva/udfs/abstract_udfs.py:74]: Can we not call these filters? Someone might think of them as filtering data

**Review by gaurav274 — CHANGES_REQUESTED** [review]

File: eva/udfs/gaussian_blur.py (hunk 2 above)
Comment (LordDarkula) [reply eva/udfs/gaussian_blur.py:51]: Isn't the resolution of the frames unknown at initialization? If not, how do I get it?

File: eva/udfs/abstract_udfs.py (hunk 4 above)
Comment (LordDarkula) [reply eva/udfs/abstract_udfs.py:74]: What should I call them? Effects?

**Comment by jarulraj** [comment]
Redundant PR.

**Pull request closed without merging**